### PR TITLE
Feature/options prod variant

### PR DESCRIPTION
**Pull request by mrterranova**
Should not contain the component Product.jsx any longer. Please check for that. Also new component "PrimaryPage.jsx" will be our replacement once I have everything set up. Please check for that as well. Thanks.

### CODE REVIEW

File: src/components/Directory.js [hunk 1]
```diff
@@ -1,6 +1,10 @@
 import { BrowserRouter as Router, Route, Switch } from "react-router-dom"; 
 import App from './App.jsx';
 import React, { Component } from 'react';
+<<<<<<< HEAD
```
Comment (NeelyAnne) [inline src/components/Directory.js:4]: Head tag

File: src/components/Directory.js [hunk 2]
```diff
@@ -11,15 +15,20 @@ import Product from "./productVariant/Product.jsx";
...
 // import CImagesAll from './CustomerImageCmpnt/CustomerImagesAll';
+>>>>>>> 0d2c2b9f41c9a8da4145bafb7c1b957cd66ac2c1
 
 class Directory extends Component {
     render(){
         return(
         <Router>
             <Switch>
                 <Route exact path="/" component={App} />
+<<<<<<< HEAD
```
Comment (NeelyAnne) [inline src/components/Directory.js:26]: head tag

File: package.json [hunk 3]
```diff
@@ -32,8 +32,18 @@
     "babel-core": "^7.0.0-0",
     "babel-plugin-transform-class-properties": "^6.24.1",
     "babel-plugin-transform-object-rest-spread": "^6.26.0",
+<<<<<<< HEAD
```
Comment (NeelyAnne) [inline package.json:35]: head tag

File: src/components/productVariant/variant-Components/AboutList.jsx [hunk 4]
```diff
@@ -1,7 +1,10 @@
 import React from 'react'; 
 
 function AboutList(props) {
+<<<<<<< HEAD
```
Comment (NeelyAnne) [inline src/components/productVariant/variant-Components/AboutList.jsx:4]: head tag

File: src/components/productVariant/variant-Components/AboutList.jsx [hunk 5]
```diff
@@ -10,11 +13,13 @@ function AboutList(props) {
 
     return(
         <div>
-            <p>
                 <ul>
+<<<<<<< HEAD
```
Comment (NeelyAnne) [inline src/components/productVariant/variant-Components/AboutList.jsx:17]: head tag

File: src/components/productVariant/variant-Components/Availability.jsx [hunk 6]
```diff
@@ -7,13 +7,22 @@ class Availability extends Component {
 
     //main pricing set-up for this section is called here
     pricingLogic = (list_price, price, shipping, ship_price, discount, inStock, message, lowstock_message) => {
+<<<<<<< HEAD
```
Comment (NeelyAnne) [inline src/components/productVariant/variant-Components/Availability.jsx:10]: head tag

File: src/components/productVariant/variant-Components/Availability.jsx [hunk 7]
```diff
@@ -29,7 +38,11 @@ class Availability extends Component {
                     </span></td></tr><tr><td id="priceblock_ourprice_lbl" className="a-color-secondary a-size-base a-text-right a-nowrap">
                          Price:
                     </td><td className="a-span12"><span id="priceblock_ourprice" className="a-size-medium a-color-price priceBlockBuyingPriceString">
+<<<<<<< HEAD
```
Comment (NeelyAnne) [inline src/components/productVariant/variant-Components/Availability.jsx:41]: head tag

File: src/components/productVariant/variant-Components/Availability.jsx [hunk 8]
```diff
@@ -48,7 +61,11 @@ class Availability extends Component {
             </div>
 
             //this is if the stock supply is running low
+<<<<<<< HEAD
```
Comment (NeelyAnne) [inline src/components/productVariant/variant-Components/Availability.jsx:64]: head tag

File: src/components/productVariant/variant-Components/Availability.jsx [hunk 9]
```diff
@@ -66,7 +83,11 @@ class Availability extends Component {
                     </div></a></div>
 
         //this is if the stock supply is empty
+<<<<<<< HEAD
```
Comment (NeelyAnne) [inline src/components/productVariant/variant-Components/Availability.jsx:86]: head tag

File: src/components/productVariant/variant-Components/Header.jsx [hunk 10]
```diff
@@ -11,10 +11,16 @@ class Header extends Component {
 
     hasTag = (tag, tag_title) => {
         if (tag === true) {
+<<<<<<< HEAD
```
Comment (NeelyAnne) [inline src/components/productVariant/variant-Components/Header.jsx:14]: head tag

Comment (mrterranova) [reply src/components/productVariant/variant-Components/Header.jsx]: should be changed now

**Pull request closed without merging**